packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -102,9 +117,18 @@ const FolderTreeItem = ({
         <span style={{ paddingLeft: `${level}rem` }}>
           <Arrange
             fitStart={(
-              <Spacer paddingLeft="medium" paddingRight="medium" isInlineBlock>
-                {itemIcon}
-              </Spacer>
+              <>
```

Comment:
You should not need the React fragment here since everything is contained within a single Spacer element at the moment.